### PR TITLE
Ws 0.4.1 generate sample data

### PR DESCRIPTION
**Pull request by mharkins99**
Sample data is being created for users, species & sightings. The data is re-added to the database tables each time the program is run from the program.cs file. This seems to work in testing as changes made to the database are overwritten when the app is re-run.

### CODE REVIEW

File: WhaleSpotting/Program.cs [hunk 1]
```diff
@@ -34,6 +37,51 @@
...
+context.WhaleSpecies.AddRange(species);
+context.SaveChanges();
+var users = SampleUsers.GetUsers();
+context.Users.AddRange(users);
+context.SaveChanges();
+var sightings = SampleSightings.GetSightings();
+var collectionOfSightings = new List<WhaleSighting>();
+var newWhaleSpecies = new WhaleSpecies();
+var newUser = new User();
+var newWhaleSighting = new WhaleSighting();
```
Comment (francescastocco) [inline WhaleSpotting/Program.cs:62]: I don't think we need to declare these 3 variables here

File: WhaleSpotting/Program.cs [hunk 2]
```diff
@@ -34,6 +37,51 @@
...
+
+foreach (var singleSighting in sightings)
```
Comment (francescastocco) [inline WhaleSpotting/Program.cs:64]: We can just call this sighting
```suggestion
foreach (var sighting in sightings)
```

File: WhaleSpotting/Program.cs [hunk 3]
```diff
@@ -34,6 +37,51 @@
 
 var app = builder.Build();
 
+//populate db with sample data if empty
```
Comment (francescastocco) [inline WhaleSpotting/Program.cs:40]: Can you add some more comments to this to explain what we're doing

**Review by francescastocco — CHANGES_REQUESTED** [review]
This looks really good!! Well done :)

File: WhaleSpotting/Program.cs (hunk 1 above)
Comment (mharkins99) [reply WhaleSpotting/Program.cs:62]: Removed variables (created within the foreach instead

File: WhaleSpotting/Program.cs (hunk 2 above)
Comment (mharkins99) [reply WhaleSpotting/Program.cs:64]: Amended all occurrences of singleSighting

**Comment by mharkins99** [comment]
Updated code based on all comments

**Review by francescastocco — APPROVED** [review]

**Pull request merged**